Module not found: Can't resolve 'utilities/collection-utils'

This one was me 🤦 Looks like my editor put in an absolute path based on the tsconfig `baseUrl`, which breaks in the transpiled JS.

```
Failed to compile.

./node_modules/andculturecode-javascript-core/dist/utilities/service-utils.js
Module not found: Can't resolve 'utilities/collection-utils' in '/Users/Brandon/SpotifySync/frontend/node_modules/andculturecode-javascript-core/dist/utilities'
```

--- a/src/utilities/service-utils.ts
+++ b/src/utilities/service-utils.ts
@@ -7,7 +7,7 @@ import { ResultRecord } from "../view-models/result-record";
 import { PagedResult } from "../interfaces/paged-result";
 import { HttpHeader } from "../enumerations/http-header";
 import { ContentType } from "../enumerations/content-type";
-import { CollectionUtils } from "utilities/collection-utils";
+import { CollectionUtils } from "./collection-utils";
 
 // -----------------------------------------------------------------------------------------
 // #region Variables

